tests/FloatingPointTest.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2013 - 2016, Roland Bock
```

Comment:
You might want to add your name and update the year.

Reply:
I hereby grant you full copyright. Consider it a gift :-)

Reply:
Thanks :-)